projects/smart-table/src/lib/components/smart-table/smart-table.component.ts

```diff
@@ -265,7 +269,23 @@ export class SmartTableComponent implements OnInit, OnDestroy {
         }
         return accumulator;
       }, []),
-      tap(filters => this.filter.next(filters)),
+      withLatestFrom(this.configuration$),
+      tap(([filters, config]) => {
+          if (config.options && config.options.resetSortOrderOnFilter === true) {
```

Comment:
kan je heel dit stuk niet beter verhuizen naar die eerste tap? als je enkel werkt met this.filters en met configuration zie ik niet waarom dat na de scan zou moeten